src/js/modules/accounts/components/ProfilePage.js

```diff
@@ -1,14 +1,16 @@
 import React from "react";
+import { bindActionCreators, compose } from "redux";
 import { connect } from "react-redux";
-import { bindActionCreators } from "redux";
+import { graphql } from "react-apollo";
+import humps from "humps";
 import { Link } from "react-router-dom";
 import injectSheet from "react-jss";
 import { Grid, Row, Col, Table } from "react-bootstrap/lib";
 import { Helmet } from "react-helmet";
 
 import { SignOutForm } from "./forms";
 import { signOut } from "../actions";
-import { getCurrentUser } from "../selectors";
+import { UserByUIDQuery } from "../../../queries";
```

Comment:
Why are you inlining queries in RoutingApp, but importing from a centralized place here? Just curious

Reply:
`UserByUIDQuery` is literally the only query where everything is shared.